src/main/java/tanukidecor/block/misc/GlobeBlock.java

```diff
@@ -43,14 +45,15 @@ public GlobeBlock(Properties pProperties) {
         super(pProperties, RotatingBlock.createShapeBuilder(SHAPE));
     }
 
-    //// METHODS ////
+    /// / METHODS ////
 
     @Override
-    public InteractionResult use(BlockState pState, Level pLevel, BlockPos pPos, Player pPlayer, InteractionHand pHand, BlockHitResult pHit) {
-        return GlobeBlockEntity.use(pState, pLevel, pPos, pPlayer, pHand, pHit);
+    protected ItemInteractionResult useItemOn(ItemStack pStack, BlockState pState, Level pLevel, BlockPos pPos, Player pPlayer, InteractionHand pHand, BlockHitResult pHitResult) {
+        InteractionResult result = GlobeBlockEntity.use(pState, pLevel, pPos, pPlayer, pHand, pHitResult);
```

Comment:
It would be easier and more consistent to refactor `GlobeBlockEntity.use` to return `ItemInteractionResult` instead of converting it after the fact.